package.json

```
@@ -6,9 +6,7 @@
   "type": "module",
   "license": "MIT",
   "packageManager": "pnpm@9.2.0",
-  "engines": {
-    "pnpm": "^9.0.0"
-  },
+
```

```suggestion
  "engines": {
    "pnpm": "^9.0.0"
  },
```

We don't want to get rid of this I think.